### ISSUE TITLE
JSError not include exception js code line and column infomation

### ISSUE THREAD

**Issue by iampennyli** [facebook/hermes#939]
## Bug Description

We have integrated the Hermes engine separately in our app and are not using React Native. We found that when running some JS code (actually bytecode) using Hermes, the JS exceptions triggered did not include information on the line and column where the exception occurred. This makes it difficult for us to locate the code causing the exception.Whether we use getStack(), getMessage(), or what()...

Hermes version: 0.12.0
React Native version (if any):  none
OS version (if any): iOS 16.0
Platform (most likely one of arm64-v8a, armeabi-v7a, x86, x86_64): arm64

**Comment by jpporto** [comment]
Hi @iampennyli,

This happens because you are stripping the debug information that the compiler generates, meaning the VM can't map bytecode offsets back to source lines. Debug info stripping can happen because

* The JS source was compiled without debug information (i.e., you passed ```-g0````when you compiled JS); or
* The compiler was told to emit output source maps (with ```--output-source-map```)

You should be using at least ```-g1``` to have stack trace information available. If you're emitting output source maps, you need to apply them to the stack traces manually.

John Paul

**Comment by iampennyli** [comment]
I have checked our command for offline generation of HBC and found out that we did not use -g(0-3). Following @jpporto's suggestion, I have used -g3 and the resulting HBC file has greatly increased in size.

We have continued testing and have found some additional information that was not mentioned previously, although the line and column numbers are still missing.

We have separated our JS codes into two packages, generating the corresponding bytecode files using -g3 for both. When loading the second bytecode file, an error occurred, but without any line or column information provided.
